djstripe/enums.py

```diff
@@ -0,0 +1,97 @@
+from enum import Enum as _Enum
+from django.utils.decorators import classproperty
+from django.utils.translation import ugettext as _
+
+
+class Enum(_Enum):
+    @classproperty
+    def choices(cls):
+        return tuple((cls.keys.get(k, k), v.value) for k, v in cls.__members__.items())
+
+    @classproperty
+    def keys(cls):
```

Comment:
``key_replacements`` idk. The name is tough because we need to be clear that the method is not returning all keys with some adjusted for sanitation, but only returning the keys that need to change